tests/requirements/oracle.txt

```diff
@@ -1 +1 @@
-cx_oracle
+cx_oracle<5.3
```

Comment:
Other requirements files are using spaces between the specifiers, e.g. `jinja2 >= 2.9.2`.